components/infobox/wikis/brawlhalla/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,131 @@
+---
+-- @Liquipedia
+-- wiki=brawlhalla
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Player = require('Module:Infobox/Person')
+local Class = require('Module:Class')
+local Variables = require('Module:Variables')
+local Math = require('Module:Math')
+local ActiveYears = require('Module:YearsActive')
+local PlayersSignatureLegends = require('Module:PlayersSignatureLegends')
+
+local _PAGENAME = mw.title.getCurrentTitle().prefixedText
+
+--role stuff tables
+local _ROLES = {
+	['admin'] = 'Admin', ['analyst'] = 'Analyst', ['coach'] = 'Coach',
+	['commentator'] = 'Commentator', ['caster'] = 'Commentator',
+	['expert'] = 'Analyst', ['host'] = 'Host', ['streamer'] = 'Streamer',
+	['interviewer'] = 'Interviewer', ['journalist'] = 'Journalist',
+	['manager'] = 'Manager', ['map maker'] = 'Map maker',
+	['observer'] = 'Observer', ['photographer'] = 'Photographer',
+	['tournament organizer'] = 'Organizer', ['organizer'] = 'Organizer',
+}
+local _CLEAN_OTHER_ROLES = {
+	['coach'] = 'Coach', ['staff'] = 'false',
+	['content producer'] = 'Content producer', ['streamer'] = 'false',
+}
+
+local _CURRENT_YEAR = tonumber(os.date('%Y'))
+local _statusStore
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _player
+
+function CustomPlayer.run(frame)
+	local player = Player(frame)
+	_args = player.args
+	_player = player
+
+	player.getStatusToStore = CustomPlayer.getStatusToStore
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.getPersonType = CustomPlayer.getPersonType
+
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+
+	return player:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'role' then return {}
```

Comment:
No role display, but saving both role1 & role2 into lpdb and using role for other stuff?

Reply:
i asked ricci if they needed role display, he said no so i kicked it
for storage and in the other functions it is needed though
(we do it like that on sc2 too)

Reply:
Ok, question is if Contributors will fill it in though if it doesn't display anything.